app/views/content/non-uk-teachers/teach-in-england-if-you-trained-overseas.md

```diff
@@ -164,19 +149,48 @@ Set up a Teaching Vacancies job alert to be updated about opportunities througho
 </p>
 
 
-## Find out what salary you can earn
+## Apply for your visa
+
+You can apply for a skilled worker visa once you have been offered a teaching job at a school which can sponsor them. Our [guidance on visas and immigration](https://www.gov.uk/government/publications/teach-in-england-if-you-qualified-outside-the-uk/teach-in-england-if-you-qualified-outside-the-uk#visas-and-immigration) explains how to apply for a skilled worker visa, what salary you will need to earn and other types of visa you may be eligible for. 
```

Comment:
Suggestion:
You can apply for a skilled worker visa once you have been offered a teaching job at a school which can sponsor them. Find out [how to apply for a skilled worker visa](https://www.gov.uk/government/publications/teach-in-england-if-you-qualified-outside-the-uk/teach-in-england-if-you-qualified-outside-the-uk#visas-and-immigration).

Reply:
Leaving as is because the other types of visa and salary issue are important details